kubernetes/mqtt2prometheus.yml

```diff
@@ -0,0 +1,66 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: mqtt2prometheus
+  namespace: mqtt
+  labels:
+    app: mqtt2prometheus
+spec:
+  type: LoadBalancer
+  ports:
+  - name: metrics
+    port: 9641
+    targetPort: 9641
+  selector:
+    app: mqtt2prometheus
+status:
+  loadBalancer: {}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mqtt2prometheus
+  namespace: mqtt
+  labels:
+    app: mqtt2prometheus
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: mqtt2prometheus
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: mqtt2prometheus
+    spec:
+      containers:
+      - args:
+        - /mqtt2prometheus
+        - -config
+        - /data/config.yaml
+        - -log-level
+        - debug
+        image: stephens/mqtt2prometheus:latest
+        imagePullPolicy: ""
+        name: mqtt2prometheus
+        ports:
+        - containerPort: 9641
+        resources: {}
+        volumeMounts:
+        - name: config-volume
+          mountPath: /data
+#          subPath: config.yaml
+      restartPolicy: Always
+      serviceAccountName: ""
+      volumes:
+      - name: config-volume
+        configMap:
+          name: mqtt2prometheus-configmap
+
+#      volumes:
```

Comment:
Please remove this commented unused resources.

Reply:
done